### PR TITLE
Added the post endpoint for the recipes

### PR DESCRIPTION
**Pull request by Naphrum**
Card: https://github.com/orgs/HTMElephant/projects/2/views/1?pane=issue&itemId=47755233

I added a post endpoint to create a recipe to then be added into the database.

### CODE REVIEW

File: routes/v1/recipes/recipes.js [hunk 1]
```diff
@@ -19,4 +19,44 @@ router.get("/", async (req, res, next) => {
...
+      user_id,
+      ingredients,
+      directions,
+      description,
+      image_url,
+      is_private,
+      category_id,
+    } = req.body;
+
+    console.log(typeof ingredients);
```
Comment (MattGVO) [inline routes/v1/recipes/recipes.js:37]: oopsie

File: routes/v1/recipes/recipes.js [hunk 2]
```diff
@@ -19,4 +19,44 @@ router.get("/", async (req, res, next) => {
...
+    const recipe = await db.recipes.insert({
+      title,
+      user_id,
+      ingredients: {
+        data: ingredients,
+        rawType: true,
+        toPostgres: (p) => {
+          return db.pgp.as.format("$1::jsonb", [JSON.stringify(p.data)]);
+        },
+      },
```
Comment (MattGVO) [inline routes/v1/recipes/recipes.js:48]: For those reviewing

This is how to insert jsonb column with massivejs

File: routes/v1/recipes/recipes.js (hunk 1 above)
Comment (Naphrum) [reply routes/v1/recipes/recipes.js:37]: Got that fixed

**Review by MattGVO — APPROVED** [review]

**Review by KyleD8n — APPROVED** [review]
Nice going man

**Pull request merged**